pysat/__init__.py

```diff
@@ -71,33 +78,51 @@
         data_dir = '/home/travis/build/pysatData'
     else:
         data_dir = ''
-    with open(os.path.join(pysat_dir, 'data_path.txt'), 'w') as f:
+    data_path_file = os.path.join(pysat_dir, 'data_path.txt')
+    with Lock(data_path_file, 'w', file_timeout) as f:
         f.write(data_dir)
+        
+        # in case of network files system
+        f.flush()
+        os.fsync(f.fileno())
+
     print(''.join(("\nHi there!  Pysat will nominally store data in the "
```

Comment:
Print statements should all be Warnings, Errors, or logging statements.

Reply:
This print is intended as a first time text to make sure users are aware they have to do something for pysat to work.